optaplanner-benchmark/src/main/resources/org/optaplanner/benchmark/impl/report/benchmarkReport.html.ftl

```diff
@@ -351,7 +379,35 @@
+                                                    <#else>
+                                                        <td><div class="dropdown">
+                                                            <span class="nav nav-pills dropdown-toggle" id="dLabel" type="button" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">
+                                                                ${singleBenchmarkResult.averageCalculateCountPerSecond!""}/s
+                                                                <span class="caret"></span>
+                                                            </span>
+                                                        <ul class="dropdown-menu" aria-labelledby="dLabel">
+                                                            <li class="dropdown-header"><strong>Individual runs</strong></li>
+                                                            <li role="separator" class="divider"></li>
+                                                            <#list singleBenchmarkResult.subSingleBenchmarkResultList as subSingleBenchmarkResult>
+                                                                <li class="dropdown-header"><strong>Run #${subSingleBenchmarkResult.getSubSingleBenchmarkIndex()}</strong></li>
+                                                                <li>${subSingleBenchmarkResult.averageCalculateCountPerSecond!""}/s</li>
+                                                            </#list>
+                                                            <#--<li role="separator" class="divider"></li>-->
+                                                            <#--<li class="dropdown-header"><strong>Average</strong></li>-->
+                                                            <#--<li>${singleBenchmarkResult.averageAverageCalculateCountPerSecond!""}/s</li>-->
+                                                            <#--<li class="dropdown-header"><strong>Standard Deviation</strong></li>-->
+                                                            <#--<li>${singleBenchmarkResult.standardDeviationStringAverageCalculateCountPerSecond!""}/s</li>-->
+                                                            <#--<li class="dropdown-header"><strong>Best</strong></li>-->
+                                                            <#--<li>${singleBenchmarkResult.bestAverageCalculateCountPerSecond!""}/s</li>-->
+                                                            <#--<li class="dropdown-header"><strong>Worst</strong></li>-->
+                                                            <#--<li>${singleBenchmarkResult.worstAverageCalculateCountPerSecond!""}/s</li>-->
+                                                            <#--<li class="dropdown-header"><strong>Median</strong></li>-->
+                                                            <#--<li>${singleBenchmarkResult.medianAverageCalculateCountPerSecond!""}/s</li>-->
```

Comment:
As a professor of mine used to say: "Copy-pasting is a common source of errors." Fixed (removed).